docs/conf.py

```diff
@@ -28,9 +28,11 @@
 # -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# extensions coming with Sphinx (named "sphinx.ext.*") or your custom
 # ones.
-extensions = ["m2r2"]
+extensions = [
```

Comment:
Stylistically, I prefer the previous version, but it doesn't even rise to the level of a nit.  Was this an automated format?

Reply:
Yes its an autoformat.  I'd been ignoring it for too long and am finally committing it.